app/views/add/PlatformOperatorAddedView.scala.html

```diff
@@ -36,14 +37,30 @@
         content = messages("platformOperatorAdded.panel.p1", viewModel.operatorId)
     ))
 
-    @if(subscriptionEmail.trim.toLowerCase != viewModel.poPrimaryContactEmail.trim.toLowerCase) {
-        <p class="govuk-body">@messages("platformOperatorAdded.p1.two.emails", viewModel.operatorName, subscriptionEmail, viewModel.poPrimaryContactEmail)</p>
-    } else {
-        <p class="govuk-body">@messages("platformOperatorAdded.p1.one.email", viewModel.operatorName, subscriptionEmail)</p>
-    }
+    <p class="govuk-body">
```

Comment:
The output here is rendered in one <pclass="govuk-body"></p>.
The page screen shot shows the content as two separate paragraphs.

Reply:
No, this is separate. There are two sentences there. Lets speak about it.